base/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM python:2.7
-
 MAINTAINER Library Simplified <info@librarysimplified.org>
 
+ARG version
```

Comment:
@daryanypl: This line allows build command be called as something like `docker build --build-arg version=1.0.7 nypl/circ-base /base`, which will set the `version` argument `'1.0.7'` and checkout tagged version 1.0.7 in the lower change.